### DISCUSSION TITLE
Translating notes and adding collection instructions

### DISCUSSION THREAD

**Discussion by MoltenCoffee** [directus/directus#4477] [Feature Requests]
Hi!

At the moment, we can set notes at several locations; In project settings for the login page, for collections, for fields and perhaps more.

I think there are two main ways we could improve the usefulness of these notes:

1. Add option to translate them, just like we do with collection name translations
2. Extend the collection note, or perhaps add another (WYSIWYG?) 'instructions' field, to allow admins to give instructions to user that add new items. We can already use the field notes to give field-based instructions, but perhaps a larger (localized) text displayed when adding a new item would help authors as well!

Interested in what you think!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
